io/jvm/src/main/scala/fs2/io/net/unixsocket/JdkUnixSockets.scala

```diff
@@ -41,28 +42,32 @@ object JdkUnixSockets {
 private[unixsocket] class JdkUnixSocketsImpl[F[_]: Files](implicit F: Async[F])
     extends UnixSockets.AsyncUnixSockets[F] {
   protected def openChannel(address: UnixSocketAddress) =
-    Resource
-      .make(F.blocking(SocketChannel.open(StandardProtocolFamily.UNIX)))(ch =>
-        F.blocking(ch.close())
-      )
-      .evalTap { ch =>
-        F.blocking(ch.connect(UnixDomainSocketAddress.of(address.path)))
-          .cancelable(F.blocking(ch.close()))
-      }
+    evalOnVirtualThreadIfAvailable(
+      Resource
+        .make(
+          F.blocking(SocketChannel.open(StandardProtocolFamily.UNIX))
+        )(ch => evalOnVirtualThreadIfAvailable(F.blocking(ch.close())))
```

Comment:
Because we are wrapping the entire `Resource` in `evalOnVirtualThreadIfAvailable`, we don't need to wrap the individual effects.

Reply:
Right, I think I missed it out when I was changing from individual wrappings to wrapping the whole `Resource`.